sdk/src/commitment_config.rs

```diff
@@ -46,4 +52,5 @@ pub enum CommitmentLevel {
     Max,
     Recent,
     Root,
+    Single,
```

Comment:
Lol!
I'm definitely open to better names, but it is actually pretty low commitment... 1 confirmation.

Note to self: update docs, when decided

Reply:
I thought it was pretty high! I guess even if a supermajority voted on a slot, it's still reasonably likely that they switch forks until "Optimistic Confirmation" is finished? cc @carllin

Reply:
Still not a big fan of "single commitment" but maybe we can rename commitment to confirmation later

Reply:
They can still easily switch forks after only 1 vote. Likelyhood depends on if the next leaders also build off this same block because they have it or it's weighted higher than other possible forks.